CMakeLists.txt

```diff
@@ -517,6 +517,41 @@ file( WRITE "${CMAKE_CURRENT_BINARY_DIR}/modules.dot" "digraph {
 execute_process( COMMAND
    dot -O -Tsvg "${CMAKE_CURRENT_BINARY_DIR}/modules.dot" )
 
+#
+# Code signing
+#
+
+cmd_option( ${_OPT}perform_codesign
+            "Perform code signing during the install step. This only works on Windows and macOS."
+            Off
+)
+
+if( ${_OPT}perform_codesign )
+   include( AudacityCodeSigning )
+endif()
+
+#
+# Packaging
+#
+
+cmd_option( ${_OPT}package_manual
+            "Package the manual along with the DMG and InnoSetup targets"
+            Off
+)
+
+# Variables, that are common for all package targets
+
+if( GIT_COMMIT_SHORT AND AUDACITY_BUILD_LEVEL LESS 2)
+   set( PACKAGE_SUFFIX "_${GIT_COMMIT_SHORT}" )
+endif()
+
+set( BASE_PACKAGE_NAME "Audacity_${AUDACITY_DIST_VERSION}${PACKAGE_SUFFIX}" )
```

Comment:
Is this variable not used anymore? Same with `PACKAGE_SUFFIX` above.

Reply:
You are correct, removed now